scripts/convert-to-platforms.sh

```
@@ -1540,58 +1541,53 @@ convert_to_cosmos() {
+
+    # Use icon URL or logo URL for icon field
+    local icon_url="${APP_ICON:-${APP_LOGO:-https://cdn.jsdelivr.net/gh/bigbeartechworld/big-bear-universal-apps/apps/_example/logo.jpg}}"
+    
+    # Use container image name/URI to create URL
+    local main_image=${APP_MAIN_IMAGE}
+    local base_url1="https://hub.docker.com/r/"
+    local base_url2="https://hub.docker.com/_/"
+    if [[ "$main_image" == ghcr.io* ]]; then
+        main_image="https://${main_image}"
+    elif [[ "$main_image" == lscr.io/linuxserver* ]]; then
+        main_image=$(echo $main_image | sed 's/lscr\.io\/linuxserver\///')
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)